src/CodeGen_Vulkan_Dev.cpp

```diff
@@ -2120,53 +2129,65 @@ void CodeGen_Vulkan_Dev::SPIRV_Emitter::visit(const Shuffle *op) {
 
     } else {
```

Comment:
If I'm not overlooking anything, it seems there is no specialized branch for simply joining (i.e., concatenating) vectors. Either put a TODO comment, make an issue, or add that branch. Falling back on the generic vector shuffle, below, seems like a performance waste.

Reply:
Tracking this here: https://github.com/halide/Halide/issues/8622